python/test/test_polars.py

```diff
@@ -543,21 +544,33 @@ def test_polars_threshold():
         .collect()
     )
 
-
+@pytest.mark.skipif(
+    os.getenv('FORCE_TEST_REPLACE_BINARY_PATH') != "1", 
+    reason="setting OPENDP_POLARS_LIB_PATH interferes with the execution of other tests"
+)
 def test_replace_binary_path():
     import os
```

Comment:
```suggestion
def test_replace_binary_path():
```
Locally, I can remove the skipif, and the redundant `import os`, and `pytest` passes for me, and it seems like it would be good to run all the tests by default? However, if you usually run the tests in parallel then introducing `FORCE_TEST_REPLACE_BINARY_PATH` may make sense...